Demo/src/main/java/com/braintreepayments/demo/PaymentButtonsFragment.java

```diff
@@ -58,13 +58,13 @@ public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, bool
                     if (!isChecked) return;
                     switch (checkedId) {
                         case R.id.button_pp_blue:
-                            payPalButton.setButtonColor(PayPalButtonColor.BLUE);
+                            payPalButton.setButtonColor(PayPalButtonColor.Blue.Default.INSTANCE);
```

Comment:
Would merchants always need to specify `.Default` for each color when setting the color programmatically?